index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css"
+  integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg=="
+  crossorigin="anonymous" />
+  <title>Student Grade Table</title>
+</head>
+<body>
+  <header class="mx-auto w-50">
+    <h1>Student Grade Table</h1>
+  </header>
+  <div class="table-responsive ">
+    <table class="table table-striped table-bordered w-50 mx-auto ">
+      <thead class="thead-light">
+        <tr>
+          <th>Student Name</th>
+          <th>Course</th>
+          <th>Grade</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+        <tr>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+        <tr>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.js"
+    integrity="sha512-WNLxfP/8cVYL9sj8Jnp6et0BkubLP31jhTG9vhL/F5uEZmg5wEzKoXp1kJslzPQWwPT1eyMiSxlKCgzHLOTOTQ=="
+    crossorigin="anonymous"></script>
+  <script src="./components/app.js"></script>
+  <script src="./components/grade-table.js"></script>
+  <script src="main.js"></script>
```

Comment:
Since `App` will use an instance of `GradeTable`, `app.js` technically depends on `grade-table.js`, even if it doesn't directly reference the `GradeTable` class. Please load other components before loading `app.js`. Since `main.js` is the top-level script that uses everything else, it should still be last.

Reply:
Also, please remove the `./` prefix from the URLs.